src/convenience/keyboard.ts

```diff
@@ -184,7 +291,18 @@ export class Keyboard {
      * @param url An HTTPS URL of a Web App to be opened with additional data
      */
     webApp(text: string, url: string) {
-        return this.add({ text, web_app: { url } });
+        return this.add(Keyboard.webApp(text, url));
+    }
+    /**
+     * Adds a new web app button. The Web App that will be launched when the
```

Comment:
```suggestion
     * Creates a new web app button. The Web App that will be launched when the
```

Like the other static methods' docs

Reply:
Whoopsie daisy, thanks a bunch